exercises/concept/tisbury-treasure-hunt/.meta/exemplar.py

```diff
@@ -1,41 +1,45 @@
+"""Functions to help Azara and Rui locate pirate treasure."""
+
+
 def get_coordinate(record):
-    """
+    """Return coordinate value from a tuple containing the treasure name, and treasure coordinate.
 
-    :param record: tuple - a (treasure, coordinate) pair.
+    :param record: tuple - with a (treasure, coordinate) pair.
     :return: str - the extracted map coordinate.
     """
 
     return record[1]
 
 
 def convert_coordinate(coordinate):
-    """
+    """Create and return a tuple containing two characters from the treasure coordinate.
 
     :param coordinate: str - a string map coordinate
-    :return:  tuple - the string coordinate seperated into its individual components.
+    :return: tuple - the string coordinate seperated into its individual components.
     """
 
     return tuple(coordinate)
 
 
 def compare_records(azara_record, rui_record):
-    """
+    """Compare two record types and determine if the coordinates match.
 
     :param azara_record: tuple - a (treasure, coordinate) pair.
-    :param rui_record: tuple - a (location, coordinate, quadrant) trio.
-    :return: bool - True if coordinates match, False otherwise.
+    :param rui_record: tuple - a (location, coordinate, quadrant) trio with a nested tuple.
```

Comment:
```suggestion
    :param rui_record: tuple - a (location, tuple(coordinate_1, coordinate_2), quadrant) trio.
```

Reply:
This is very frustrating.  I do not want to add in Type Hints, but I also don't want these docstrings to become paragraphs.  Gah.  I may have made this worse.